src/common.js

```diff
@@ -20,7 +20,7 @@ const inProduction =
     verifyMinified.name !== "verifyMinified"
 
 let autoFreeze = !inProduction
-let useProxies = typeof Proxy !== "undefined"
+let useProxies = typeof Reflect !== "undefined"
```

Comment:
please check both Reflect and Proxy :)

Reply:
To guard against rare environments? According to MDN, `Reflect` is never available where `Proxy` isn't on the major browsers.

Reply:
@mweststrate This would work?
```
// Use Proxy only when both Proxy and Reflect is supported.
let useProxies = typeof Proxy !== undefined && typeof Reflect !== undefined
```

@aleclarson are you planning to fix this or do you want me to create a new PR with this fix? :)